vault_cli/cli.py

```diff
@@ -285,7 +285,7 @@ def delete(client_obj: client.VaultClientBase, name: str) -> None:
     "--path",
     multiple=True,
     required=True,
-    help="Folder or single item. Pass several times to load multiple values",
+    help="Folder or single item. Pass several times to load multiple values. You can use --path mypath=prefix if you want to change the generated names of the environment variables",
```

Comment:
the "prefix" only applies to folders. Maybe we should say "name" ?

Reply:
it applies to folders OR keys, so I don't know what name to choose :)

Reply:
maybe the help text needs to be a bit more precise then ? I'm affraid "prefix" would be confusing for secret, we would imagine there's the prefix + the secret name ?

Reply:
I have added a section in the doc of "env", tell me if it's ok

Reply:
:+1: